components/kubearchive/base/kustomization.yaml

```
@@ -132,6 +132,12 @@ patches:
                 runAsNonRoot: true
               ports:
                 - containerPort: 8081
+              resources:
+                resources:
```

shouldn't this second `resources` be `requests`

```suggestion
                requests:
```

---

You are totally right, thanks!

---

Weirdly, kubelinter does not flag it as an error.